book/004-functions-classes.md

```diff
@@ -472,36 +472,34 @@ get isPaid() {
 }
 ```
 
-Now if we want to see if an invoice is paid or not, we can just run `Invoice.isPaid()`.
-
-Under the hood, the `isPaid` method is added to the object/classes's `prototype` chain, which gives it access to `this`.
+Now if we want to see if an invoice is paid or not, we can just run `Invoice.isPaid` (note the lack of parenthesis). Under the hood, `isPaid` is a function, but it is accessed like a property.
 
 ```js
-SomeInvoice.isPaid(); // => Boolean
+SomeInvoice.isPaid; // => Boolean
 ```
 
-Note that there's no actual `isPaid` attribute on the an instance of `Invoice`. `isPaid` grabbing and returning `true` or `false` depending on the value of `this.receivedOn`.
+Note that there's no actual `isPaid` property on an instance of `Invoice`. `isPaid` grabbing and returning `true` or `false` depending on the value of `this.receivedOn`.
 
-Now let's say our client is awesome and paid us for our work (hooray!). Let's create a `setter` to set the `receivedOn` date.
+Let's create a `setter` for isPaid as well. We'll assume that if we set isPaid to true, we received payment today.
 
 ```js
 /**
   Pays an invoice and sets the date the invoice was paid.
-  @param  {Date}    date - The date the invoice was paid on
-  @return {Invoice}
+  @param  {paid}    paid - interpreted as a boolean
```

Comment:
This whole example might be nice, but it's starting to devalue what a setter can do and returning `this`. 

One thing I failed to mention is that by returning `this`, we're able to chain methods onto the object. I don't think this example needs to be changed, so if you can revert this change back to what it was, I think it would be better.